src/Amazon.AspNetCore.DataProtection.SSM/Amazon.AspNetCore.DataProtection.SSM.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
Since .NET Standard seems to be dead I think we should change this to `netcoreapp3.1`. .NET 3.1 will still be are minimum for sometime while users are still transitioning to .NET 6.

Reply:
I will enable multi-targeting for net3.1, net50, and net60 as well.

Reply:
I do however, recommending semantic major change to 3.0.0.

Reply:
Agreed this will be a major version bump since we are dropping platforms.